app/src/main/java/com/breadwallet/tools/manager/TxManager.java

```diff
@@ -81,8 +81,7 @@ public void onLongItemClick(View view, int position) {
 
             }
         }));
-        if (adapter == null)
-            adapter = new TransactionListAdapter(app, null);
+        adapter = new TransactionListAdapter(app, null);
```

Comment:
need to recreate the `TransactionListAdapter` after change the language, since the `TxManager.init` only called in one place

Reply:
I guess just always do it because there is not performance hit when the user changes languages (when they are picking) @andhikayuana ?

Reply:
sure, actually, if we use an existing instance of TransactionListAdapter, it will not recreate the layout because the locale is bound to the context.